pkg/config.go

```diff
@@ -91,14 +91,13 @@ func getConfig(lang string, configFile string) (*config, error) {
 	return conf, nil
 }
 
-func defaultConf(lang string) (*config, error) {
+func defaultConf(lang string) (conf *config, err error) {
 	switch lang {
 	case golangLanguage:
 		yamlFile, _ := langTemplate.ReadFile("templates/golang.yaml")
-		conf := &config{}
```

Comment:
You need to keep this line. If you do not, conf is not initialized with an address, i.e. `conf=nil` and the Unmarshal command fails. That is why the CI fails

Reply:
https://play.golang.org/p/vfAaZoOnXkm